src/main/java/cubicchunks/world/cube/Cube.java

```diff
@@ -333,8 +457,9 @@ public void getEntitiesWithinAABBForEntity(Entity excluded, AxisAlignedBB queryB
 
 	public void tickCube(boolean tryToTickFaster) {
 		if (!this.isInitialLightingDone && this.isPopulated) {
-			this.tryDoFirstLight();
+			this.tryDoFirstLight(); //TODO: Vary icky light population code! REMOVE IT!
```

Comment:
Spelling was in the comment, in this case. Even if it's a TODO, I think it should still be spelled correctly